### PR TITLE
Updating Playground-ready from main

### PR DESCRIPTION
**Pull request by asirota**
# Pull Request

## What changed?

added PHP unit tests, fixed a bug

## Why did it change?

because new PHP unit tests have been added

## Did you fix any specific issues?

#88 

## CERTIFICATION

By opening this pull request, I do agree to abide by
the [CODE OF CONDUCT](https://github.com/aspirepress/.github/CODE_OF_CONDUCT.md) and be bound by the terms
of the [Contribution Guidelines](https://github.com/aspirepress/.github/CONTRIBUTING.md) in effect on the date and time
of my contribution as proven by the
revision information in GitHub. I also agree that any previous contributions shall be deemed subject to the terms of the
version in effect on the date and time of this pull request, or any future revisions for pull requests I may submit.
Further, I certify that this work is my own, is original, does not violate the intellectual property of any other person
or entity, and I am not violating any license agreements or contracts I have with any person or entity. Finally, I agree
that this code may be licensed under any license deemed appropraite by AspirePress, including but not
limited to open source, closed source, proprietary or custom licenses, and that such license terms neither violate my
rights or my copyright to this code.

### CODE REVIEW

**Comment by asirota** [comment]
I closed the previous PR and opened this one and resolved conflict in VS code using merge editor there, please review PR so I can update playground-ready branch with latest changes.

**Review by chuckadams — APPROVED** [review]

**Comment by chuckadams** [comment]
Alas, as I don't have write access to the repo, my review approval doesn't unlock any merge to a protected main branch. 🤷

**Comment by asirota** [comment]
Yeah there's a resolve conflict that's in one file and I have to do it on command line and resolve in merge code editor.

**Pull request merged**
